lib/rules/media-feature-name-case/index.js

```diff
@@ -1,13 +1,13 @@
 const { createPlugin } = require('stylelint')
 const atRuleParamIndex = require('stylelint/lib/utils/atRuleParamIndex')
 const isCustomMediaQuery = require('stylelint/lib/utils/isCustomMediaQuery')
-const isRangeContextMediaFeature = require('stylelint/lib/utils/isRangeContextMediaFeature')
 const isStandardSyntaxMediaFeatureName = require('stylelint/lib/utils/isStandardSyntaxMediaFeatureName')
 const mediaParser = require('postcss-media-query-parser').default
-const rangeContextNodeParser = require('stylelint/lib/rules/rangeContextNodeParser')
 const report = require('stylelint/lib/utils/report')
 const ruleMessages = require('stylelint/lib/utils/ruleMessages')
 const validateOptions = require('stylelint/lib/utils/validateOptions')
+const rangeContextNodeParser = require('../../stylelint-utils/rangeContextNodeParser')
```

Comment:
nit

just utils as the name for the folder would be fine
i.e. if I ever wanted to add an util specific to this plugin I wouldn't want them separated